src/data_readers/data_reader_jag_conduit.cpp

```diff
@@ -353,26 +358,24 @@ bool data_reader_jag_conduit::load_conduit_node(const size_t i, const std::strin
     }
   }
 
-  conduit::relay::io::hdf5_read(h, path, node);
+  h.read(path, node);
```

Comment:
Are we mixing in the new PR about conduit I/O nodes?  If so then line 324 probably needs to change.

Reply:
no, it's using the old/existing hdf5 interface, since the PR for IOHandle isn't ready. I had some problems with it, and stopped working on in until the preload code is finished -- because I thought it was most urgent to get that working.

Reply:
Okay, then I am confused by these lines of code that seem to be using the newer IOHandle?